ci/linkcheck.sh

```diff
@@ -27,5 +27,3 @@ else # running locally
 
   echo "Checking files changed in $COMMIT_RANGE: $CHANGED_FILES"
 fi
-
-exec mdbook-linkcheck $FLAGS -- $TRAVIS_BUILD_DIR
```

Comment:
Wait, why was this whole line removed? Without this line, this script is useless.

Reply:
Ouch, my bad, fixed.